dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/BaseTraceService.java

```diff
@@ -503,6 +505,12 @@ public static void applyJsonFields(String value) {
                     auditMap.put(entry[0], entry[1]);
                     valueFound = true;
                 }
+                //Check if mapped value is an extension
+                if (entry[0].startsWith("ext_")) {
+                    messageMap.put(entry[0], entry[1]);
+                    traceMap.put(entry[0], entry[1]);
+                    valueFound = true;
+                }
```

Comment:
need to have a corresponding section in the entry.length == 3 section.  Also need to avoid letting people specify ext_ mappings for things other than message and trace in that section.